src/pages/manage/movies/MovieForm.tsx

```diff
@@ -567,10 +600,12 @@ const MovieForm = () => {
     ];
   }, [t]);
 
+  const title = t(`movies.create.title`);
+
   return (
     <Page>
       <Page.Title spacing={3} alignItems="center">
-        {t(`movies.create.title`)}
+        {title}
       </Page.Title>
 
```

Comment:
Any reason why you moved away from the inline translation?

Reply:
Because I want to move it to props in a later PR.